components/scream/scripts/gather_all_data.py

```diff
@@ -32,6 +32,9 @@
     "summit" : (["module purge", "module load cmake/3.15.2 gcc/6.4.0 spectrum-mpi/10.3.0.1-20190611 cuda/10.1.168 python/3.6.6-anaconda3-5.3.0"],
                 "$(which mpicxx)",
                 "bsub -I -q batch -W 0:30 -P cli115 -nnodes 1"),
+    "cori"   : (["eval $(../../cime/scripts/Tools/get_case_env)", "export CTEST_PARALLEL_LEVEL=68", "export OMP_NUM_THREADS=68"],
```

Comment:
Note: slick use of get_case_env to leverage CIME to set up the scream standalone env!

Reply:
Do you mind briefly explaining how that works? Just in case we have to do some maintenance.